docs/deploy.md

```diff
@@ -548,6 +548,169 @@ This exposes a JSON-RPC interface to the Hardhat Network. Use the address, **htt
+
+![deployedcontract](/img/docs/deploy/deployedcontracts.png)
+
+### Interact with a Contract Instance
+
+Interacting with your smart contract can be a simple transaction, such as sending some amount of Ether to trigger an action, unlike a generic smart contract method call, where a transaction will change the state of the EVM.
+
+Perform the following steps to interact with run your smart contract:
+
+1. In the **Deployed Contract** pane, click on the **right arrow** next to the filename to expand. In this example, the filename is TESTCONTRACT. The functions of the deployed contract are displayed.
+
+![threebuttons](/img/docs/deploy/threebuttons.png)
+
+Upon expanding the pane, your contract instance displays three actions that corresponds to the functions:
+
+**SetNP (orange button)** - This is a non-payable function, where it is not possible to send value (ZEN) to the contract.
+
+**SetP (red button)** - This is a payable function that sends value (ZEN) to the contract instance.
+
+**get (blue button)** - The get function does not execute a transaction, therefore it does not change the state (the value variable in your contract) of your contract instance. However, it is a view function and a return value is shown.
+
+2. Click on the **down arrow** next to the **setP** field to expand the function.
+
+![expandfunction](/img/docs/deploy/expandfunction.png)
+
+#### Non-Payable Functions
+
+For **non-payable functions**, DO NOT specify a value other than zero (0). Any value more than 0 is not accepted by the smart contract and the transaction will fail.
+
+![nonpayfunction](/img/docs/deploy/nonpayablefunction.png)
+
+#### Payable Functions
+
+For **payable functions**, you can specify an amount of ZEN in the Value field to be paid to the smart contract. A successful execution of the transaction subtracts this value amount including the required fees for a *normal* smart contract method call.
+
+![payfunction](/img/docs/deploy/payablefunction.png)
+
+If any issues arise, the Remix IDE will provide a status notification before confirming the operation.
+
+![confirmation](/img/docs/deploy/confirmation.png)
+
+3. In the expanded function (**setP**), enter a **desired value** in the **value** field.  
+
+   **Note:** Initially, the fields, **setP** and **setNP** display *uint256*. This indicates that it is expecting an unsigned integer (256 bit) type to store non-negative integers.  
+
+4. Click **transact**. The Terminal panel displays a success status for the interaction with your smart contract.
+
+![sucessmessage](/img/docs/deploy/successmessage.png)
+
+### Interaction with Sample Contract 
+
+To illustrate interaction with smart contracts, the **EonWorkspace** is created in Remix. This workspace provides a set of script files to perform specific tasks, such as displaying information or executing a transaction within the blockchain.
```

Comment:
I wanted to check - is the "EonWorkspace" file something the user can import?
Or do they need to create it themselves?

Reply:
Sounds good @Ramone-HL .
Do you know when we can be expecting that? Without it, referencing the .js files doesn't have much value.
Could you also add a ticket to track this work? Both for the link and updating to the docs